tests/cupy_tests/core_tests/test_raw.py

```diff
@@ -341,12 +342,20 @@ def use_temporary_cache_dir():
                 yield path
 
 
+_in_memory_cache = os.environ.get('CUPY_CACHE_IN_MEMORY')
+
+
 @testing.parameterize(*testing.product({
     'backend': ('nvrtc', 'nvcc'),
+    'in_memory': ('1', '0'),
 }))
```

Comment:
How about writing without `testing.product`?
```suggestion
@testing.parameterize(
    {'backend': 'nvrtc', 'in_memory': False},
    {'backend': 'nvrtc', 'in_memory': True},
    {'backend': 'nvcc', 'in_memory': False},
)
```

Reply:
This is much nicer! Thanks.